crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -244,7 +244,6 @@ where
     Self: Sized,
 {
     /// Convert into a [`SystemConfigs`].
-    #[doc(hidden)]
```

Comment:
This only makes it visible in the docs; it's already accessible in user code (just tested it in one of my Bevy projects).